docs/security_vulnerability_disclosure/user_tooling.md

```diff
@@ -0,0 +1,78 @@
+---
+title: User Tooling
+layout: page
+previous:
+  url: vulnerability_definition
+  title: Vulnerability Definition
+next:
+  url: process
+  title: Process
+---
+
+Users of BEAM languages like Erlang and Elixir have access to several tools to
```

Comment:
```suggestion
Users of BEAM languages, like Erlang and Elixir, have access to several tools to
```

Reply:
Additionally, I'd recommend "such as" instead of "like".